buttons/buttons.css

```diff
@@ -1270,3 +1270,46 @@ border-radius: 25px;
 }
 
 /* blurry btn end */
+
+
+/* ANIKET btn start */
+.aniket-sinha-btn {
```

Comment:
change as it is told in readme.md 
yourname-button-index   here index is missing

Reply:
also at all place where u have used